python/src/aiconfig/scripts/rage/rage.py

```diff
@@ -0,0 +1,60 @@
+import os
+import lastmile_utils.lib.core.api as core_utils
+import sys
+import time
+import itertools
+import numpy as np
+from result import Ok, Result
+
+logger = core_utils.get_logger(__name__)
+
+
+class RageConfig(core_utils.Record):
+    log_level: str | int = "WARNING"
+
+
+def rage(config: RageConfig) -> Result[None, str]:
+    logger.setLevel(config.log_level)
+    print("\n\n\n\n............\n\n")
+    print("Please open an issue! :) Here are some tips on how to do that:")
+    for logfile in ["editor_flask_server.log", "aiconfig.log"]:
+        print()
+        if os.path.exists(logfile):
+            print(f"Found {logfile}! Please include its contents in your bug report.")
+            print(f"Full path: {os.path.abspath(logfile)}")
+        else:
+            print(f"No {logfile} found. This might be another bug :)")
+            print("For now, please include your terminal output in your bug report.")
+
+    print("\nPlease run the following commands and also include their output:")
+    print("\nwhich pip; which pip3; which python; which python3; pip3 list | grep aiconfig; python --version; python3 --version")
+
+    print("\nPlease open an issue here: https://github.com/lastmile-ai/aiconfig/issues/")
+    print("Our sincerest gratitude. We will comment on the issue as soon as possible.")
```

Comment:
I think `sincerest gratitute` is kind of overkill, especially with `Checking your flipping server logs` the tone isn't really consistent

Reply:
I think we should probably remove the trolling